extensions/amp-ad/0.1/amp-ad.css

```diff
@@ -48,9 +48,13 @@ amp-embed iframe {
   padding: 2px 4px;
 }
 
-amp-ad[type=adsense] > iframe {
+amp-ad[data-a4a-upgrade-type] > iframe {
```

Comment:
Also, this is going to apply to _all_ FF slots, regardless of ad network, yes?  Is that okay?  I mean, I know that we need it for AdSense/Doubleclick, but these slots will be used by non-Google ad networks as well.